org.eclipse.scout.sdk.core/src/main/java/org/eclipse/scout/sdk/core/util/Strings.java

```diff
@@ -1747,4 +1750,63 @@ public static String removeSuffix(String string, CharSequence suffix, boolean is
     }
     return string;
   }
+
+  /**
+   * Removes the given prefix from the given string. If the given string does not start with given prefix (case
+   * sensitive), the string is returned unchanged.
+   *
+   * @param string
+   *          The string in which the prefix should be removed. May be {@code null}, then {@code null} is returned.
+   * @param prefix
+   *          The prefix to remove (case sensitive) or {@code null} if nothing should be removed.
+   * @return The string with the prefix removed.
+   */
+  public static String removePrefix(String string, CharSequence prefix) {
+    return removePrefix(string, prefix, true);
+  }
+
+  /**
+   * Removes the given prefix from the given string. If the given string does not start with given prefix, the string is
+   * returned unchanged.
+   *
+   * @param string
+   *          The string in which the prefix should be removed. May be {@code null}, then {@code null} is returned.
+   * @param prefix
+   *          The prefix to remove or {@code null} if nothing should be removed.
+   * @param isCaseSensitive
+   *          Specifies if the prefix should match case sensitive or not.
+   * @return The string with the prefix removed.
+   */
+  public static String removePrefix(String string, CharSequence prefix, boolean isCaseSensitive) {
+    if (startsWith(string, prefix, isCaseSensitive) && prefix.length() > 0) {
```

Comment:
second check "prefix.length() > 0": is this necessary? string.substring() does this check already, doesn't it?